cmake/scripts/common/PrepareEnv.cmake

```diff
@@ -46,9 +46,9 @@ if(CORE_SYSTEM_NAME STREQUAL android)
   endif()
 elseif(CORE_SYSTEM_NAME STREQUAL darwin_embedded)
   if (CPU MATCHES armv7)
```

Comment:
I would have added `set(PLATFORM_TAG ${CORE_PLATFORM_NAME})` above this line and kept everything else as it was, to be more consistent with the other blocks of this if/elseif statement.

Reply:
Cool, will change that.